lint-staged.config.js

```diff
@@ -2,7 +2,8 @@
 
 module.exports = {
   "*.{js,js.flow,ts}": "eslint --fix --report-unused-disable-directives",
-  "*.md": "prettier --write",
+  "*.{md,mdx}": "prettier --write",
+  "*.mdx": "eslint --report-unused-disable-directives",
```

Comment:
To apply Prettier (if indeed it does work):
```suggestion
  "*.mdx": "eslint --fix --report-unused-disable-directives",
```

Reply:
The reason I'd not done this was cause I wasn't sure that the fixes from remark-lint would be applied properly. I guess we can try it out and see.